webapp/src/components/common/dialogs/DigestDialog/types.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2025, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+export interface DigestMatrixType {
```

Comment:
You can rename it `DigestMatrix`

Reply:
DigestMatrix is already taken by the component, I will rename it something else as the suffix "Type" should be avoided, `DigestMatrixData` or `DigestMatrixConfig`